src/Resources/Deals.php

```diff
@@ -138,13 +138,16 @@ function associateWithContact($dealId, $contactIds)
     
     /**
      * @param int $contactId
+     * @param array $params Optional parameters ['limit', 'offset']
      * @return mixed
      */
-    function associatedWithContact($contactId)
+    function associatedWithContact($contactId, array $params = [])
```

Comment:
i haven't paid enouhh attention to all PRs, but I'll need to go through and remove array typehints at some point.

lots of people use collection objects, and other objects that have array access for things, but php doesn't really have a good typehint to cover those cases, unfortunately.